TechnicoServer/TechnicoDomain/Models/OwnerType.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace TechnicoDomain.Models;
```

Comment:
The OwnerType enum could be placed inside the Owner class for better organization, as it seems tightly coupled to it.